pages/verification/index.tsx

```diff
@@ -48,10 +53,13 @@ export default function Verification(props: PageProps) {
         })
     }
 
-    const sendAnonymousMessage = (anonymousToken: string) => {
+    const sendAnonymousMessage = (anonymousToken: string, optionalParams: SendAnonymousMessageOptionalParams = {}) => {
+        const {replaceRoute = false, withLoading = false} = optionalParams
+
         const member = localStorage.getItem(MessageEnum.targetUser);
         const message = localStorage.getItem(MessageEnum.tempMessage);
         apiAnonymous.defaults.headers.common['Authorization'] = `Bearer ${anonymousToken}`;
+        if(withLoading) setIsLoading(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendAnonymousMessage` function now accepts an optional parameter, which is a good way to extend the function's functionality without breaking existing code. However, there's no error handling for when `localStorage.getItem()` returns null or undefined. This could lead to unexpected behavior if these values are not set in local storage.

Consider adding error handling or checks for these values before using them.
```diff
+         const member = localStorage.getItem(MessageEnum.targetUser);
+         const message = localStorage.getItem(MessageEnum.tempMessage);
+         if (!member || !message) {
+             console.error('Member or message not found in local storage');
+             return;
+         }
          apiAnonymous.defaults.headers.common['Authorization'] = `Bearer ${anonymousToken}`;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->